jscomp/core/js_name_of_module_id.ml

```diff
@@ -25,20 +25,6 @@
 let (=)  (x : int) (y:float) = assert false
 *)
 
-#ifdef BS_BROWSER
-
-let string_of_module_id_in_browser (x : Lam_module_ident.t) =
-  match x.kind with
-  | External { name } -> name
-  | Runtime | Ml -> "./stdlib/" ^ String.uncapitalize_ascii (Lam_module_ident.name x) ^ ".js"
```

Comment:
This is no longer needed as stdlib is consumed as any other package (#602 was removing it already)